ui/src/app/models/permission-level.ts

```diff
@@ -0,0 +1,7 @@
+// Levels of authorization / permission for CDRs.
+
+export enum PermissionLevel {
+  Public = 1,
```

Comment:
I don't think there's going to be "public" for the workbench, FWIW. Just registered and controlled.

If you're not a registered user, I expect we'll kick you to some screen saying "no workbench for you."

(This is assuming all the public-facing storefront stuff is elsewhere.)